recipe/meta.yaml

```
@@ -7,24 +7,26 @@ package:
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7f07f997eb19aedb414d2afe780744321033154a36cb46788913097d9912fff9
+  sha256: 07481e7346d33416857e1d7a41d6cd6a8261b62314d73c7edc9e45e612fa3ac9
 
 build:
   number: 0
   #skip s390x as lack of nodejs and cartopy dependencies
-  skip: True  # [py<310 or s390x]
+  skip: true  # [py<310 or s390x]
```

```suggestion
  skip: true  # [py<310]
```
You can drop s390x as we don't support it anymore